llvm/test/Analysis/LoopAccessAnalysis/long-pointer-distance.ll

```diff
@@ -0,0 +1,73 @@
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -passes='print<access-info>' -disable-output %s 2>&1 | FileCheck %s
+
+define void @forward_i128_offset_difference(ptr %this, i128 %loop.limit) {
+; CHECK-LABEL: 'forward_i128_offset_difference'
+; CHECK-NEXT:    loop:
+; CHECK-NEXT:      Memory dependences are safe
+; CHECK-NEXT:      Dependences:
+; CHECK-NEXT:        Forward:
+; CHECK-NEXT:            store i64 1, ptr %c, align 8 ->
+; CHECK-NEXT:            store i64 2, ptr %d, align 8
```

Comment:
Does this test actually cover the case where `trySExtValue`fails and because of that we cannot identify the dependency? In that case, should it be Unknown?

Reply:
From the commit message: "In the first test, since getTypeSizeInBits is not equipped to handle type-sizes over 64-bits, the constant distance flowing through LAA is -1." -- the value we get from SCEV is wrapped already, as SCEV uses getTypeSizeInBits, and trySExtValue doesn't have a chance to fail.